impl/src/main/java/org/jboss/weld/bean/builtin/InstanceImpl.java

```diff
@@ -101,9 +101,17 @@ public T get() {
         if (bean != null) {
             return getBeanInstance(bean);
         } else if (isUnsatisfied()) {
-            throw BeanManagerLogger.LOG.unresolvableElement("Type: " + getType() + "; Qualifiers: " + getQualifiers());
+            throw BeanManagerLogger.LOG.injectionPointHasUnsatisfiedDependencies(ip,
+                    Formats.formatAnnotations(ip.getQualifiers()),
+                    Formats.formatInjectionPointType(ip.getType()),
+                    Formats.formatAsStackTraceElement(ip),
```

Comment:
Stack trace element should not be there. Remember that a real injection point is not always available - i.e. `Instance.select()`.